mtp_noms_ops/assets-src/stylesheets/views/_security-form-fields.scss

```diff
@@ -93,3 +93,35 @@ select[disabled] {
     }
   }
 }
+
+#filter-senders {
+  .column-one-third {
+    .mtp-info-box {
+      padding: 1em;
+
```

Comment:
can you put in a `@include media(mobile) {...}` rule to add a margin; it touches the search button otherwise

Reply:
👍 didn't like it either but the prototype didn't have the margin, I think Gavin wouldn't mind so I'll add it.